businessRules/ideas.js

```diff
@@ -81,9 +72,8 @@ function rejectedIdeasList() {
 }
```

Comment:
It's not clear from the name `pendingIdeasList`, `approvedIdeasList` and `rejectedIdeasList` whether they are setters or getters. Can you improve their names please?

Reply:
A getter. It returns a list. Fixed.